pytest_mozwebqa/pytest_mozwebqa.py

```diff
@@ -22,9 +23,12 @@ class DeferPlugin(object):
     """Simple plugin to defer pytest-html hook functions."""
 
     def pytest_html_environment(self, config):
-        server = config.option.sauce_labs_credentials_file and \
-            'Sauce Labs' or 'http://%s:%s' % (config.option.host,
-                                              config.option.port)
+        driver = config.option.driver
+        if hasattr(cloud, driver.lower()):
```

Comment:
Neat! I didn't know you could use `hasattr` to look for objects in a namespace like this.